pkg/controllers/kluctldeployment_controller_status.go

```diff
@@ -35,6 +37,20 @@ func (r *KluctlDeploymentReconciler) patchReadyCondition(ctx context.Context, ob
 
 // patchFail returns the original error + patchErr if required
 func (r *KluctlDeploymentReconciler) patchFail(ctx context.Context, obj *kluctlv1.KluctlDeployment, reason string, err error) error {
+	var lastDeployResult *result.CommandResultSummary
+	if obj.Status.LastDeployResult != nil {
+		parseErr := yaml.ReadYamlBytes(obj.Status.LastDeployResult.Raw, &lastDeployResult)
+		if parseErr != nil {
+			log.Info(fmt.Sprintf("Failed to parse last deploy result: %s", parseErr.Error()))
+		}
+	}
+
+	startTime := time.Now().Unix()
```

Comment:
Same as above, please don't use int64 to represent time. Also, please only set the metric when parsing from above succeeded (basically move the Set call into a `parseErr == nil` if block.